e2e_test/source/kafka.slt

```diff
@@ -293,9 +293,9 @@ select id, first_name, last_name, email from s8;
 1005 add add2 add
 
 query IITFFBTT
-select id, sequence_id, name, score, avg_score, is_lasted, entrance_date, birthday from s9;
+select id, sequence_id, name, score, avg_score, is_lasted, entrance_date, birthday, passed from s9;
 ----
-32 64 str_value 32 64 t 1970-01-01 1970-01-01 00:00:00
+32 64 str_value 32 64 t 1970-01-01 1970-01-01 00:00:00 1 mon 1 day 00:00:01
```

Comment:
no change in test_data?

Reply:
`scripts/source/test_data/avro_bin.1` has been modified to add the `passed` data.